dipy/workflows/align.py

```diff
@@ -190,3 +197,460 @@ def run(self, static_files, moving_files,
                          .format(moved_centroids_file))
             save_trk(moved_centroids_file, centroids_moved, affine=np.eye(4),
                      header=static_header)
+
+
+class ImageRegistrationFlow(Workflow):
+    """
+    The registration workflow is organized as a collection of different
+    functions. The user can intend to use only one type of registration
+    (such as center of mass or rigid body registration only).
+
+    Alternatively, a registration can be done in a progressive manner.
+    For example, using affine registration with progressive set to 'True'
+    will involve center of mass, translation, rigid body and full affine
+    registration. Whereas, when progressive is False the registration will
+    include only center of mass and affine registration. The progressive
+    registration will be slower but will improve the quality.
+
+    This can be controlled by using the progressive flag (True by default).
+    """
+
+    def perform_transformation(self, static, static_grid2world, moving,
+                               moving_grid2world,
+                               affreg, params0, transform, affine):
+
+        """ Function to apply the transformation.
+
+        Parameters
+        ----------
+        static : array, shape (S, R, C) or (R, C)
```

Comment:
You might want to make the notation here consistent with the changes here: https://github.com/nipy/dipy/pull/1735/files#diff-74c995598e7ee365915e537084234a8bR962

Reply:
Thanks